pkg/api/v1/helper/helpers.go

```diff
@@ -433,6 +433,28 @@ func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
 	return ""
 }
 
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeClaimVolumeMode(claim *v1.PersistentVolumeClaim) v1.PersistentVolumeMode {
+	defaultMode := v1.PersistentVolumeFilesystem
```

Comment:
This line is not needed, you can just return filesystem at the end.

Reply:
yep